content/articles/spatial_mgrs/index.Rmd

```diff
@@ -0,0 +1,427 @@
+ggplot(data = countries) +
+  geom_sf(fill = "grey97", colour = "grey50") + 
+  grids$geom_3035[1:14] +
+  coord_sf(datum = "EPSG:4326", xlim = c(2.5e6, 6e6), ylim = c(1.5e6, 4.5e6)) +
+  scale_x_continuous(breaks = seq(-24, 48, 6)) +
+  scale_y_continuous(breaks = seq(0, 80, 8)) +
+  theme(panel.grid = element_line(colour = "blue"))
+```
+
+Below the grid zone level it is still the UTM grid pattern that drives the further subdivision in MGRS between 80°S and 84°N, down to the required precision, just as in other grid systems.
+However an important difference at this level is that MGRS uses an _**alphanumeric grid reference**_ instead of a numeric one as in civil grids.
+In MGRS, Latin characters are used in referencing both the applicable grid zone and the first subdivision thereof, as follows.
+
+An MGRS 'grid reference' (sometimes called MGRS coordinate) is defined as:
+
+```
+[Grid Zone Designation] [Grid Square ID] [Truncated integer X-coordinate] [Truncated integer Y-coordinate]
+```
+
+An example is `31U ES 56914 61249` at the 1 m precision, which resolves to `31U ES 56 61` at 1 km precision.
+
+The spaces can be omitted or kept, but preferably omitted in a computational environment.
+The integer coordinates are truncated depending on the required precision, up to _zero_ digits (precision 100 km), with the _maximum_ of 5 digits per coordinate corresponding to a precision of 1 meter (i.e. the unit of the UTM coordinate system).
+Consequently, MGRS grid references need less digits per coordinate for 1 m precision than UTM grid references, and this is seen as an advantage of MGRS.
+
+```{r utm-grid-latbands-belgium, fig.cap="The 'U' latitude band goes from 48°N to 56°N. Two resulting MGRS grid zones are fully shown here: grid zones 31U (with most of Belgium) and 32U (with most of Germany)."}
+ggplot(data = countries) +
+  geom_sf(fill = "grey97", colour = "grey50") + 
+  grids$geom_3035[5:8] +
+  coord_sf(datum = "EPSG:4326", xlim = c(3.5e6, 4.5e6), ylim = c(2.7e6, 3.7e6)) +
+  scale_x_continuous(breaks = seq(-24, 48, 6)) +
+  scale_y_continuous(breaks = seq(0, 80, 8)) +
+  theme(panel.grid = element_line(colour = "blue"))
+```
+
+In the UTM area of MGRS (between 80°S and 84°N), an MGRS grid reference is built as follows:
+
+- The **Grid Zone Designation** (GZD), e.g. `31U`, combines the UTM zone (31) with a letter code (U) that corresponds to a latitude band of generally 8° wide; hence each intersection defines the _**MGRS grid zone**_.
+- Each grid zone is further subdivided into _**grid squares**_ (cells) of size 100 km, using the _UTM coordinate system_ to define the cell border coordinates as multiples of 100 000 m.
+The **Grid Square ID** is a code of 2 characters that is unique _within_ the grid zone, e.g. `ES`, with 'E' referring to a column and 'S' to a row.
```

Comment:
There exist two different schemes for the **Grid Square ID** ([Wikipedia.org](https://en.wikipedia.org/wiki/Military_Grid_Reference_System#100,000-meter_square_identification)). Perhaps you should mention this here.

Reply:
Yes, it's well explained in the specifications by NGA. I had previously chosen to not go into the lettering details. I still don't really, but I have added the main principles in the text, and mentioned the two lettering schemes in a footnote. It certainly helps understanding (at least to get an idea of the complexity / type of rules).

d3ced7c7